mayaUSD.mod.template

```diff
@@ -1,32 +1,37 @@
-+ USD ${USD_VERSION} ${PXR_USD_LOCATION}
++ USD ${USD_VERSION} ${USD_INSTALL_LOCATION}
+icons: 
+plug-ins: 
+presets: 
+scripts: 
+resources: 
```

Comment:
Use Maya module specific tags to not add anything specific for these paths since USD is not a Maya plugin and doesn't need Maya to know about its icons/presets/etc.